packages/table/src/Row/InternalRowWithRT.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { Fragment, ReactElement, ReactNode } from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+import InternalCellWithRT from '../Cell/InternalCellWithRT';
+import { LeafygreenTableRow } from '../useLeafygreenTable';
+
+import InternalRowBase from './InternalRowBase';
+import {
+  expandedContentStyles,
+  nestedBgStyles,
+  nestedBorderTopStyles,
+} from './Row.styles';
+import { InternalRowWithRTProps } from './Row.types';
+
+const InternalRowWithRT = <T extends unknown>({
+  children,
+  className,
+  row,
+  virtualRow,
+  disabled,
+  ...rest
+}: InternalRowWithRTProps<T>) => {
+  const { theme } = useDarkMode();
+  const isNestedRowParent = row.depth === 0 && row.getIsExpanded();
+  const isNestedRowOrParent = row.getIsExpanded() || row.depth > 0;
+  const ExpandedContentRowProp = row?.original.renderExpandedContent;
```

Comment:
nit: lc E?

Reply:
it's a function component which is why it's capitalized to indicate that

Reply:
also this comment was trippy - I thought it was a terrible typo before I decoded it lol